### PR TITLE
fix: Brazilian translation

### PR DESCRIPTION
**Pull request by thomasmazon**
## What does this PR do?

Fixes # (issue)

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/public/static/locales/pt-BR/common.json [hunk 1]
```diff
@@ -1940,4 +1940,25 @@
   "insights_subtitle": "Veja insights de reserva em seus eventos",
   "custom_plan": "Plano personalizado",
   "ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS": "↑↑↑↑↑↑↑↑↑↑↑↑↑ Adicione suas novas strings aqui em cima ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑"
```
Comment (emrysal) [inline apps/web/public/static/locales/pt-BR/common.json:1942]: Eh, how did this happen? - please uncommit the yarn.lock and move the fixes above this line :)

File: apps/web/public/static/locales/pt-BR/common.json [hunk 2]
```diff
@@ -1939,5 +1939,26 @@
...
+  "label_15_min": "15 minutes",
+  "label_30_min": "30 minutes",
+  "label_60_min": "60 minutes",
+  "nameless_team": "Equipe Sem Nome",
+  "event_host": "Responsável",
+  "event_attendee": "Participante",
+  "nameless": "Sem Nome",
+  "default_meeting": "Agenda Pessoa",
+  "default_slug": "agenda-pessoal",
+  "phone_call": "Attendee Phone Number",
```
Comment (keithwillcode) [inline apps/web/public/static/locales/pt-BR/common.json:1957]: These next 3 should be translated, no?

**Review by keithwillcode — CHANGES_REQUESTED** [review]
Could you revert the yarn.lock change please?

File: apps/web/public/static/locales/pt-BR/common.json (hunk 2 above)
Comment (keithwillcode) [reply apps/web/public/static/locales/pt-BR/common.json:1957]: @thomasmazon thoughts here?

Comment (thomasmazon) [reply apps/web/public/static/locales/pt-BR/common.json:1957]: About those 4, just miss on translation.

**Pull request closed without merging**